include/libopencm3/stm32/h7/irq.json

```diff
@@ -0,0 +1,155 @@
+{
+    "irqs": [
+        "nvic_wwdg",
+        "pvd",
+        "tamp_stamp",
+        "rtc_wkup",
+        "flash",
+        "rcc",
+        "exti0",
+        "exti1",
+        "exti2",
+        "exti3",
+        "exti4",
+        "dma1_stream0",
+        "dma1_stream1",
+        "dma1_stream2",
+        "dma1_stream3",
+        "dma1_stream4",
+        "dma1_stream5",
+        "dma1_stream6",
+        "adc",
+        "fdcan1_it0",
+        "fdcan2_it0",
+        "fdcan1_it1",
+        "fdcan2_it2",
+        "exti9_5",
+        "tim1_brk_tim9",
+        "tim1_up_tim10",
+        "tim1_trg_com_tim11",
+        "tim1_cc",
+        "tim2",
+        "tim3",
+        "tim4",
+        "i2c1_ev",
+        "i2c1_er",
+        "i2c2_ev",
+        "i2c2_er",
+        "spi1",
+        "spi2",
+        "usart1",
+        "usart2",
+        "usart3",
+        "exti15_10",
+        "rtc_alarm",
+        "reserved1", 
+        "tim8_brk_tim12",
+        "tim8_up_tim13",
+        "tim8_trg_com_tim14",
+        "tim8_cc",
+        "dma1_stream7",
+        "fsmc",
+        "sdmmc1",
+        "tim5",
+        "spi3",
+        "uart4",
+        "uart5",
+        "tim6_dac",
+        "tim7",
+        "dma2_stream0",
+        "dma2_stream1",
+        "dma2_stream2",
+        "dma2_stream3",
+        "dma2_stream4",
+        "eth",
+        "eth_wkup",
+        "fdcan_cal",
+        "cm7_sev",
+        "reserved2",
+        "reserved3", 
+        "reserved4",
+        "dma2_stream5",
+        "dma2_stream6",
+        "dma2_stream7",
+        "usart6",
+        "i2c3_ev",
+        "i2c3_er",
+        "otg_hs_ep1_out",
+        "otg_hs_ep1_in",
+        "otg_hs_wkup",
+        "otg_hs",
+        "dcmi",
+        "cryp",
+        "hash_rng",
+        "fpu",
+        "uart7",
+        "uart8",
+        "spi4",
+        "spi5",
+        "spi6",
+        "sai1",
+        "lcd_tft",
+        "lcd_tft_err",
+        "dma2d",
+        "sai2",
+        "quadspi",
+        "lp_timer1",
+        "hdmi_cec",
+        "i2c4_ev",
+        "i2c4_er",
+        "spdifrx",
+        "otg_fs_ep1_out",
+        "otg_fs_ep1_in",
+        "otg_fs_wkup",
+        "otg_fs",
+        "dmamux1_ov",
+        "hrtim1_mst",
+        "hrtim1_tima",
+        "hrtim1_timb",
+        "hrtim1_timc",
+        "hrtim1_timd",
+        "hrtim1_time",
+        "hrtim1_flt",
+        "sai3",
```

Comment:
What h7 are you working on here?  there's a series of DFSDM interrupts here in RM0433?